src/components/lib/esmap.js

```diff
@@ -452,13 +455,13 @@ function renderNodeControl(g, data, ref, layerId){
       PubSub.publish("showEditNodeDialog", { "object": pointData, "index": spliceIndex, "layer": pointData.layer }, ref.svg.node());
     })
     .on('mouseover', function (event, d) {
-      if(d.meta.template){
-        var text = renderTemplate(d.meta.template, {"d": d, "self": d });
-      } else {
-        var text = `<p><b>${ d.meta.displayName || d.name}</b></p>
-        <p><b>In Volume: </b> ${d.inValue}</p>
-        <p><b>Out Volume: </b> ${d.outValue}</p>`;
-      }
+      let text;
+      const template = ref.options.enableCustomNodeTooltip ? ref.options.customNodeTooltip : defaultNodeTooltip;
+
+      text = renderTemplate(template, {...d, "self": d });
+      text = text.replaceAll(/\s+/g, " ")
+        .replaceAll("><", "> <")
+        .trim();
```

Comment:
what's the intention of all this "replaceAll"?

Reply:
This was intended to help the readability of debugging output while working on tests. This is now removed.